mono/mini/aot-compiler.c

```diff
@@ -1815,11 +1820,25 @@ arch_emit_plt_entry (MonoAotCompile *acfg, const char *got_symbol, int offset, i
 		/* Used by mono_aot_get_plt_info_offset */
 		emit_int32 (acfg, info_offset);
 #elif defined(TARGET_AMD64)
+#ifdef MONO_ARCH_CODE_EXEC_ONLY
```

Comment:
Please use runtime if over compile-time ifdef.
Some day there should be one mon.exe with all AOT backends.